cyber-resilience-sig/minutes/2025-03-03-mom-cyber-resillience-sig.md

```diff
@@ -12,3 +12,91 @@
 | FAQ Task Force | Tobie & All | 5 |
 | Inventory Task Force | Maxim & Tobie | 5 |
 | AOB | | 5 |
+
+## Notes 
+<details>
+<summary><b>Participants </b></summary>
+
+* Juan Rico (Eclipse Foundation)  
+* Tobie Langel (UnlockOpen/Eclipse Foundation)  
+* Daniel Thompson (Tauri / Commons Conservancy)  
+* Seth Larson (Python Software Foundation)  
+* Timo Perälä (Nokia)  
+* Jonne Soininen (Nokia)  
+* Roman Zhukov (Red Hat)  
+* Jan Westerkamp (iJUG)  
+* Marty Haught (Ruby Central)  
+* Andrew Katz (Orcro Limited/Bristows LLP)  
+* Dick Brooks (Business Cyber Guardian)  
+* Maxim Baele, (OWASP)  
+* Olle E. Johansson (Edvina/OWASP)  
+* Jeremy Stanley (Spec Committee, OpenInfra, SPI)  
+* Victor Roland (OBEO)  
+* Anne Dickison (FreeBSD Foundation)  
+* Vicky Risk (ISC)  
+* Alistair Woodman (EEF)  
+* Daniel Dilger (Mercedes-Benz Tech Innovation)  
+* Salve J. Nilsen (CPANSec)  
+* Simon Phipps (SWH)  
+* Dirk-Willem van Gulik (Apache Software Foundation)  
+* Thierry Carrez (OpenInfra Foundation)  
+* Seth Larson  
+* Ria Ferrel Schalnat  
+* Pierre Pochery (FreeBSD Foundation)  
+* Lucia Lanfri (CEN/CENELEC)  
+* Alberto Pianon (Array)
+</details>
+
+### Stakeholder consultation workshop for CRA Standardization
+
+Lucia Lanfri from CEN/CENELEC presented two opportunities aligned with the areas ORC WG is addressing:
+- Rapporteur position open for PT3 \- reach out to Lucia if there is interest.  
+- Stan4CR \- Stakeholder consultation workshop  
+  - Webinar next week  
+  - Workshop 8th of April in Copenhagen objectives:  
+    - Vulnerability handling  
+    - Lines 1 and 15 of the Standardization request  
+    - Identification of gaps in WG9  
+  - No public minutes available  
+  - Tobie will participate and then share outcomes with the WG.
+  - [Lucia's Presentation STAN4CR](https://github.com/user-attachments/files/19148895/ORC_presentation_STAN4CR.pdf)
```

Comment:
Can we add the presentation to the repository itself (so people have a copy when they clone the repo)? It makes us less dependent on GitHub.